src/lib/librang90/rkco_gg.f90

```diff
@@ -330,8 +330,12 @@ SUBROUTINE RKCO_GG (JA,JB,CORD,INCOR,ICOLBREI)
               IF (NQ1(K1) .LE. 1) CYCLE
             END IF
             JA2 = JA1
-            IF(JA.NE.JB) THEN
-              IF(IDQG.NE.2) THEN
+            IF(JA /= JB) THEN
+              IF(NPEEL == 1 .AND. NQ1(K1) == NQ2(K1)) THEN
+!
+!               TARP TU PACIU BUSENU
```

Comment:
Could we translate this comment?

Reply:
I think it's fine, should be possible to understand from the if statements.